packages/cli/scripts/preinstall.js

```diff
@@ -1,21 +1,33 @@
+const process = require('node:process')
 // @ts-check
 const { drawBox } = require('@prisma/internals/dist/utils/drawBox')
 
 export function main() {
-  // process.version (e.g. `v16.0.0`)
-  printMessageAndExitIfUnsupportedNodeVersion(process.version)
+  // Node.js version, without the `v` prefix (e.g. `16.13.0`)
+  const nodeVersion = process.versions.node
+
+  printMessageAndExitIfUnsupportedNodeVersion(nodeVersion)
+}
+
+function extractSemanticVersionParts(version) {
+  return version
+    .split('.')
+    .slice(0, 2) // only major and minor version
+    .map((v) => parseInt(v, 10))
 }
 
 export function printMessageAndExitIfUnsupportedNodeVersion(nodeVersion) {
-  const nodeVersionAsParts = nodeVersion.split('.')
-  // `.slice(1)` removes `v` from `v16`
-  const nodeMajorVersion = parseInt(nodeVersionAsParts[0].slice(1))
-  const nodeMinorVersion = parseInt(nodeVersionAsParts[1])
+  const MIN_NODE_VERSION = '16.13'
```

Comment:
Optionally:
```suggestion
  const MIN_NODE_VERSION = '16.13.0'
```

or 
```suggestion
  const MIN_NODE_VERSION = '16.13.x'
```

but this would require changing the test snapshots as well.

Reply:
We document `16.13` in
https://www.prisma.io/docs/reference/system-requirements

It's ok without the patch version I think.